src/tests/ftest/util/soak_utils.py

```diff
@@ -429,19 +430,26 @@ def launch_vmd_identify_check(self, name, results, args):
     else:
         uuids = uuid_list
     self.log.info("VMD device UUIDs: %s", uuids)
+    host_uuids = get_storage_query_device_uuids(self.dmg_command)
+    for host, uuid_dict in host_uuids.items():
+        uuid_list = sorted(uuid_dict.keys())
+        self.log.info("Devices on hosts %s: %s", host, uuid_list)
+        # Now check whether the random uuid belongs to a particular host.
+        for uuid in uuids:
+            if uuid in uuid_list:
+                dmg.hostlist = host
+                # Blink led
+                dmg.storage_led_identify(ids=uuid, timeout=2)
+                # check if led is blinking
+                result = dmg.storage_led_check(ids=uuid)
+                # determine if leds are blinking as expected
+                for value in list(result['response']['host_storage_map'].values()):
+                    if value['storage']['smd_info']['devices']:
+                        for device in value['storage']['smd_info']['devices']:
+                            if device['ctrlr']['led_state'] != "QUICK_BLINK":
+                                failing_vmd.append([device['ctrlr']['pci_addr'], value['hosts']])
+                                status = False
```

Comment:
`status` is never defined above so I expect `"status": status,` below could raise an exception.

Reply:
status is defined on the latest update.
`    # pylint: disable=too-many-nested-blocks
    status = True

I accidentally removed it from the original code.